lib/lookup.json

```
@@ -153,6 +153,12 @@
     "prefix": "v",
     "maintainers": "dougwilson"
   },
+  "fastify": {
+    "maintainers": ["mcollina"],
+    "prefix": "v",
+    "skip": ["s390x", "aix", "aix72-ppc64"],
```

```suggestion
    "skip": ["s390x", "aix"],
```

You don't need `aix72-ppc64` if `aix` is also in this set. Skips are only considered for `citgm-all` -- if you're running `citgm <module>` CITGM assumes you wanted to run for that module.

---

Ah awesome, so I don't need a full green run.